tools/state-viewer/src/cli.rs

```diff
@@ -148,6 +152,26 @@ impl DumpStateRedisCmd {
     }
 }
 
+#[derive(Parser)]
+pub struct DumpTxCmd {
+    /// Specify which block to dump transactions for.
+    #[clap(long)]
+    height: BlockHeight,
+    /// List of account IDs to dump.
+    /// If not set, all account IDs will be dumped.
+    #[clap(long)]
+    account_ids: Option<Vec<AccountId>>,
+}
+
+impl DumpTxCmd {
+    pub fn run(self, home_dir: &Path, near_config: NearConfig, store: Store) {
+        match dump_tx(self.height, home_dir, near_config, store, self.account_ids.as_ref()) {
+            Ok(_) => (),
```

Comment:
you can do 'expect' instead of the match.

dump_tx(...).expect("Failed to dump transaction...")